1-mandatory/2-people-i-know.js

```diff
@@ -378,22 +378,46 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+for (var i = 0; i < friends.length; i++){
+  if (friends[i].age >= 35){
+    thirtyFiveOrOlder.push(friends[i]);
+    // thirtyFiveOrOlder.push(friends[i].name.first);
+  }
+};
 
```

Comment:
Well done 
Consider: You could also do it with the use of filter method
let thirtyFiveOrOlder = friends.filter(friend => friend.age > 35);